frontend/src/lib/components/Milestone.svelte

```diff
@@ -124,105 +114,154 @@ function selectAnswer(answer: number | undefined) {
 		nextMilestone();
 	}
 }
+
+async function setup() {
+	const response = await getCurrentMilestoneAnswerSession({
+		path: { child_id: currentChild.id },
+	});
+
+	if (response.error) {
+		console.log("Error when retrieving milestone answer session");
+		showAlert = true;
+		alertMessage =
+			$_("milestone.alertMessageRetrieving") + " " + response.error.detail;
+		milestoneAnswerSession = undefined;
+	} else {
+		milestoneAnswerSession = response.data;
+	}
+}
+
+let milestoneAnswerSession = $state(
+	null as MilestoneAnswerSessionPublic | null | undefined,
+);
+let currentMilestoneIndex = $state(contentStore.milestoneIndex);
+let currentMilestone = $state(undefined as MilestonePublic | undefined);
+let selectedAnswer = $derived(
+	milestoneAnswerSession?.answers?.[`${currentMilestone?.id}`]?.answer,
+);
+let showAlert = $state(false);
+let alertMessage = $state("");
+let autoGoToNextMilestone = $state(false);
+let currentImageIndex = $state(0);
+const promise = setup();
+const breadcrumbdata = $derived([
+	{
+		label: currentChild.name,
+		onclick: () => {
+			activeTabChildren.set("childrenRegistration");
+		},
+		symbol: UserSettingsOutline,
+	},
+	{
+		label: $_("milestone.groupOverviewLabel"),
+		onclick: () => {
+			activeTabChildren.set("milestoneGroup");
+		},
+		symbol: RectangleListOutline,
+	},
+	{
+		label: contentStore.milestoneGroupData.text[$locale].title,
+		onclick: () => {
+			activeTabChildren.set("milestoneOverview");
+		},
+		symbol: GridOutline,
+	},
+	{
+		label:
+			String(currentMilestoneIndex + 1) +
+			"/" +
+			String(contentStore.milestoneGroupData.milestones.length),
+		symbol: EditOutline,
+	},
+]);
 </script>
 
+{#await promise}
+<p>{$_("userData.loadingMessage")}</p>
+{:then}
 <div
-	class="border-1 flex flex-col border border-gray-200 bg-white shadow md:max-w-7xl md:rounded-lg dark:border-gray-700 dark:bg-gray-800"
+	class="mx-auto flex flex-col p-4 md:rounded-t-lg"
```

Comment:
not sure if it is this line or elsewhere but the styling has some issues for me:

![image](https://github.com/user-attachments/assets/b6085161-0c34-4e23-a84c-ab102dfc09e2)

Reply:
I didn´t manage to reproduce this so far. was the image expecially large or anything?

Reply:
ok, so I experimented with the [`Carousel`](https://flowbite-svelte.com/docs/components/carousel#Component_data)  but didn´t get it work, and eventually gave up. I hope  this works now with the modified tailwind classes and I didn´t overlook some cases.